conf.yaml

```diff
@@ -127,8 +127,8 @@ contents:
             prefix:     en/welcome-to-elastic
             current:    *stackcurrent
             index:      welcome-to-elastic/index.asciidoc
-            branches:   [ 8.4 ]
-            live:       [ 8.4 ]
+            branches:   [ 8.5 ]
+            live:       [ 8.5 ]
```

Comment:
We can get rid of this line. If `live` is missing, the build assumes all the `branches` are live. So having both with the same value isn't needed